tests/views/index.jade

```diff
@@ -1,12 +1,14 @@
 extends layout
 
 append scripts
+    script(src='../node_modules/babel-polyfill/dist/polyfill.js')

```

Comment:
Did I merge something with `Array.prototype.includes` without realising? Whoops!

Well yes, that makes sensse then.

Keep in mind that karma doesn't hit the HTML file, so it doesn't need all the polyfills in here too.